circleguard/enums.py

```diff
@@ -267,6 +272,52 @@ class Mod():
              V2, TD, # we stop caring about order after this point
              FI, RD, CN ,TP, K1, K2, K3, K4, K5, K6, K7, K8, K9, CO, MR]
 
+    # same as ORDER but with NM included
+    ALL = [NM, EZ, HD, HT, DT, _NC, HR, FL, NF, SD, _PF, RX, AP, SO, AT,
+           V2, TD, FI, RD, CN ,TP, K1, K2, K3, K4, K5, K6, K7, K8, K9, CO, MR]
+
+    def __init__(self, value):
+        if type(value) is int:
```

Comment:
wouldn't be `isinstance(value, int)` be better?

Reply:
yes, meant to change it to that (though in practice it doesn't matter since nobody really subclasses primitives)